clients/privacy-center/components/consent/NoticeDrivenConsent.tsx

```diff
@@ -199,7 +199,7 @@ const NoticeDrivenConsent = () => {
       preferences,
       user_geography: region,
       privacy_experience_id: experience?.id,
-      method: ConsentMethod.BUTTON,
+      method: ConsentMethod.SAVE,
```

Comment:
This is a subtle-but-useful bugfix to make the Privacy Center record the method as "save" instead of the more generic "button" option, which we deprecated.

Unrelated to this change but worth fixing 👍